settings.yml

```diff
@@ -30,7 +50,7 @@ variables:
   ######################################
 
   bicepRegistryDoPublish: true # Set to true, if you would like to publish module templates to a bicep registry
-  bicepRegistryName: adpsxxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
+  bicepRegistryName: adpzu3bxazacrx001 # The name of the bicep registry (ACR) to publish to. If it does not exist, it will be created.
```

Comment:
Please also revert the other settings

Reply:
I think we don't / cannot replace token in our pipeline yml file as it is loaded before we could process it (at least in Azure DevOps).
Fortunately, for the ACR it's not so important as it's a deployment that sticks (and hence has low to no risks of conflicts).